.github/workflows/tests.yml

```diff
@@ -15,6 +15,7 @@ jobs:
         strategy:
             matrix:
                 php-version: ["7.1", "7.2", "7.3", "7.4", "8.0", "8.1"]
+                extensions: ["none, mbstring", "none, apcu, mbstring"]
```

Comment:
You will need to add more extensions, see:
https://github.com/phpmyadmin/sql-parser/blob/0147fc9a96228156adf0ffea7772ce7c47a20c22/.github/workflows/tests.yml#L37

Reply:
Yeah. Working out all that it needs looked hard - I've just changed it to `[":apcu, mbstring", "apcu, mbstring"]`

Reply:
Perfect, thank you for the attention you did bring on this concern